test/unit/modules/signatures.js

```diff
@@ -14,66 +14,226 @@
 
 'use strict';
 
+const application = require('../../common/application');
+const transactionTypes = require('../../../helpers/transaction_types');
+const ApiError = require('../../../helpers/api_error');
+
 /* eslint-disable mocha/no-pending-tests */
+/* eslint-disable no-unused-vars */
 describe('signatures', () => {
+	let library;
+
+	before(done => {
+		application.init(
+			{ sandbox: { name: 'lisk_test_modules_signatures' } },
+			(err, scope) => {
+				library = scope;
+				done(err);
+			}
+		);
+	});
+
 	describe('isLoaded', () => {
-		it('should return true if modules exists');
+		it('should return true if modules exists', done => {
+			library.rewiredModules.signatures.__set__('modules', {
+				accounts: library.modules.accounts,
+				transactions: library.modules.transactions,
+				transport: library.modules.transport,
+			});
```

Comment:
I would avoid setting rewiredModules in test scope. Also this will mutate the library object for upcoming tests which can cause having non-atomic tests.

Reply:
please don't forget to use // Arrange, Act, Assert comments when logic gets a bit complicated inside the test.

Reply:
Addressed it